rootppl/dists/dists_gpu.cuh

```diff
@@ -84,4 +84,8 @@ __device__ floating_t gamma(curandState* randState, floating_t k, floating_t the
 }
 */
 
-#endif
\ No newline at end of file
+
+
+
+
+#endif
```

Comment:
Restore original white space here (no actual change to this file)

Reply:
done